Relativity.Processing.SDK.md

```diff
@@ -4,20 +4,41 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version
+* Updated DFV to properly populate Processing Deletion fields.
+* Ported Post Publish Delete performance improvement changes from mayapple.
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Fix adding index to zprocessingdochistory table.
+* Advanced Picker pagination support in files tab.
+* Removing toggle for Processing Profile Inclusion/Exclusion feature.
+* ARM Processing: Do not throw exception for duplicate cancel requests for canceled or canceling workspaces and Fix Processing Exists readiness check defect.
+* Replaced remaining WCF endpoints with Kepler endpoints.

```

Comment:
This should be removed as it's specific to Invariant infrastructure and isn't something the RAP exposes.